lib/systems/inspect.nix

```diff
@@ -11,6 +11,7 @@ rec {
     isi686         = { cpu = cpuTypes.i686; };
     isx86_64       = { cpu = cpuTypes.x86_64; };
     isPowerPC      = { cpu = cpuTypes.powerpc; };
+    isppc64le      = { cpu = cpuTypes.powerpc64le; };
```

Comment:
Hmm since this shorthand exists just at the nix level (in this PR) at least, can you capitalize it?